crap-advisor-frontend/src/core/components/Map/index.jsx

```diff
@@ -1,29 +1,39 @@
-import React from 'react';
-import {YMaps, Map as YMap, ZoomControl, Placemark} from '@pbe/react-yandex-maps';
+import React, {useEffect, useState} from "react";
 import "./index.css"
-import restroomImage from '../../../assets/restroom.png'
+import {MapContainer, TileLayer, ZoomControl} from "react-leaflet";
+import RestroomMarker from "../RestroomMarker";
+
+const apiUrl = "http://localhost:8082"
+const defaultPosition = [53.21176, 50.18394]
+const defaultZoom = 13
 
 const Map = () => {
+    const [restrooms, setRestrooms] = useState([])
+
+    useEffect(() => {
+        fetch(apiUrl + "/restroom/all")
+        .then((response) => response.json())
+        .then((data) => {
+            setRestrooms(data)
+            console.log(`Received ${data.length} restrooms`)
+        })
+    }, [])
+
     return (
-        <YMaps class={"map"}>
-            <YMap
-                defaultState={{center: [52.75, 52.75], zoom: 5}}
-                height="100vh"
-                width="100vw"
-                modules={['control.ZoomControl']}
-            >
-                <ZoomControl options={{position: {right: "30px", bottom: "50px"}}}/>
-                <Placemark
-                    geometry={[55.68, 37.73]}
-                    options={{
-                        iconLayout: "default#image",
-                        iconImageHref: restroomImage,
-                        iconImageSize: [32, 32],
-                        iconContentOffset: [32, 32]
-                    }}
-                />
-            </YMap>
-        </YMaps>
+        <MapContainer className={"map"} center={defaultPosition} zoom={defaultZoom} zoomControl={false}>
```

Comment:
Do not write className prop in curly brakes

Reply:
Ok, this is my ide doing that